test/integration/TestClient.ts

```diff
@@ -29,17 +30,27 @@ const TESTS_FOLDER_PATH = path.join(getUserConfigFolder(), 'bpmn-studio-tests');
 const DATABASE_PATH = path.join(TESTS_FOLDER_PATH, 'process_engine_databases');
 const SAVE_DIAGRAM_DIR = path.join(TESTS_FOLDER_PATH, 'saved_diagrams');
 const VISIBLE_TIMEOUT = 40000;
-const REMOVE_COMMAND = process.platform === 'win32' ? 'rmdir /s /q' : 'rm -rf';
+const REMOVE_COMMAND_DIR = process.platform === 'win32' ? 'rmdir /s /q' : 'rm -rf';
+const REMOVE_COMMAND_FILE = process.platform === 'win32' ? 'del /f' : 'rm -rf';
```

Comment:
The r should not be necessary, but it does not really matter.

```suggestion
const REMOVE_COMMAND_FILE = process.platform === 'win32' ? 'del /f' : 'rm -f';
```